src/lib/header-bidding/prebid/prebid.ts

```diff
@@ -493,9 +494,10 @@ const requestBids = async (
 	}
 
 	const adUnits = await onConsent()
-		.then((consentState) => {
+		.then(async (consentState) => {
 			// calculate this once before mapping over
-			const pageTargeting = getPageTargeting(consentState);
+			const isSignedIn = await isUserLoggedInOktaRefactor();
+			const pageTargeting = getPageTargeting(consentState, isSignedIn);
```

Comment:
No action required here, but some ideas for future:

It seems like a pattern is emerging where we need to retrieve a consent state and a signed in status (both asynchronously) before we can get the page targeting. From a data-flow perspective, it might be clearer to

1. Construct the page targeting object **once** (I'm sure we recreate it a whole bunch of times at the moment) and store that value, awaiting both consent state and signed in status when we do so.
2. Pass the page targeting object into all the places that need it, rather than calling for it each time.

Reply:
Yes I like the idea of constructing the page-targeting object once, I think that would be neater and reduce a bit of complexity. Definitely one to think about incorporating moving forward!